### PR TITLE
Discovery View

### LINKED ISSUE #663: Implement discovery front end
Designs: https://redpen.io/p/gp122b0c5b39e0a85d

* Replace near me with newest
* used "featured" flag (boolean) for firebase storage

### PR DESCRIPTION
**Pull request by gvn**
Resolves #663 

R? @k88hudson

### CODE REVIEW

File: views/discover/index.js [hunk 1]
```diff
@@ -1,12 +1,56 @@
...
+
+        this.showFeatured();
+    },
+    methods: {
+        showNewest: function () {
+            var self = this;
+
+            self.mode = 'Newest';
+
+            fb
```
Comment (k88hudson) [inline views/discover/index.js]: can you change the query so guest apps are omitted?

File: views/discover/index.js [hunk 2]
```diff
@@ -1,12 +1,56 @@
...
+                .child('apps')
+                .limitToLast(10)
```
Comment (k88hudson) [inline views/discover/index.js]: let's store this value in a var at the top since we want it consistent for both

File: views/discover/index.js [hunk 3]
```diff
@@ -1,12 +1,56 @@
...
+        showFeatured: function () {
+            var self = this;
+
+            self.mode = 'Featured';
+
+            fb
+                .child('apps/')
+                .orderByChild('isFeatured')
+                .equalTo(true)
+                .limitToLast(10)
```
Comment (k88hudson) [inline views/discover/index.js]: see above

File: views/discover/index.js [hunk 4]
```diff
@@ -1,12 +1,56 @@
 var view = require('../../lib/view');
-var fakeDiscovery = require('../../lib/fake-discovery');
+var Firebase = require('firebase');
+var config = require('../../config/index.js');
+
+var fb = new Firebase(config.FIREBASE_URL);
```
Comment (k88hudson) [inline views/discover/index.js]: personally i would use `this.$root.storage._firebase` in your functions

Comment (k88hudson) [inline views/discover/index.js]: no need to create a new instance

**Comment by k88hudson** [comment]
@gvn a couple of changes but generally this is working quite well

File: views/discover/index.js (hunk 1 above)
Comment (gvn) [inline views/discover/index.js]: I'm making a new ticket for this since it's more complex than it should be. :imp:

**Comment by k88hudson** [comment]
R+

**Pull request merged**